apps/health_campaign_field_worker_app/lib/pages/individual_details.dart

```diff
@@ -173,7 +197,7 @@ class _IndividualDetailsPageState
         'individualName': FormControl<String>(value: ''),
         'idType': FormControl<String>(value: ''),
         'idNumber': FormControl<String>(value: ''),
-        'dob': FormControl<String>(value: ''),
+        'dob': FormControl<DateTime>(),
```

Comment:
is there support for date without time in flutter? We should use it for dob if available